torch/csrc/jit/codegen/cuda/kernel.h

```diff
@@ -150,6 +154,19 @@ class TORCH_CUDA_CU_API Kernel final : public Fusion {
   WarpPaddedParallelInfo warp_padded_parallel_info_;
 };
 
+//! A special debugging proxy for Kernel.
```

Comment:
Can this go directly in the test file then or does it need to be here?

Reply:
I'd think it needs to be somewhere in the same library since it must be a friend of `Kernel`. The test file is linked with the library, so I don't think it's possible to define the class in the test file.